Sources/ProjectSpec/Decoding.swift

```diff
@@ -4,18 +4,36 @@ import PathKit
 import Yams
 
 extension Dictionary where Key: JSONKey {
-
-    public func json<T: NamedJSONDictionaryConvertible>(atKeyPath keyPath: JSONUtilities.KeyPath, invalidItemBehaviour: InvalidItemBehaviour<T> = .remove) throws -> [T] {
+    public func json<T: NamedJSONDictionaryConvertible>(atKeyPath keyPath: JSONUtilities.KeyPath, invalidItemBehaviour: InvalidItemBehaviour<T> = .remove, parallel: Bool = false) throws -> [T] {
```

Comment:
parallelizing the json parsing

Reply:
fyi if we used zippyjson for the json parsing, even single-threaded it would be a lot (maybe 10x?) faster. but parallelizing is a simple win for this commit